src/features/todo/Todo.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { createTask, isCompleted, editTask, deleteTask } from './todoSlice'
+import { useState } from 'react'
+
+export default function Todo() {
+    const [discription, setDiscribtion] = useState('')
+    const [editText, setEditText] = useState('')
+    const [isTaskToEdit, setIsTaskToEdit] = useState(-1)
+    const dispatch = useDispatch()
+    const todo = useSelector(state => state.todo)
+
+    console.log(todo)
+    
+    const handleSubmit = (e)=> {
+        e.preventDefault();
+        dispatch(createTask(discription))
+        setDiscribtion('')
+    }
+    const handleDelete =(id) => () =>{
+        dispatch(deleteTask(id))
+    }
+    const handleComplet = id => () =>{
+        dispatch(isCompleted(id))
+    }
+    const handleEdit = (id, description) => () =>{
+       setEditText(description) 
+       setIsTaskToEdit(id)
+
+    }
+    const handleupdate = e => () => {
```

Comment:
Some comment applies for this function and the 2 above it as the comment I left before this one.